src/test/java/org/fundacionjala/converter/model/parameter/multimedia/VideoParameterTest.java

```
@@ -72,13 +82,47 @@ public void testSetVideoCodecInvalidThrowsInvalidDataException() {
     }
     @Test 
     public void testSetCodecInvalidThrowsInvalidDataException() {
+        parameter.setFormat(".mp4");
+        parameter.setVideoCodec("h264");
         try{
-            parameter.setCodec("libmp3");
+            parameter.setCodec("mov");
             parameter.validate();
             fail();
         }
         catch (InvalidDataException e){
             assertThat(e.getMessage(), is("Invalid codec"));
         }
     }
+    @Test 
+    public void testSetFramesInvalidThrowsNotNullOrEmptyException() {
+        parameter.setFormat(".gif");
+        parameter.setVideoCodec("h264");
+        parameter.setDuration(duration);
+        parameter.setStart(start);
+        parameter.setSecondsToOutput(secondsToOutput);
+        parameter.setControlLoop(controlLoop);
+        try{
+            parameter.setFrames(null);
+            parameter.validate();
+        }
+        catch (InvalidDataException e){
+            assertThat(e.getMessage(), is("Invalid data on field = frames"));
+        }
```

instead of using catch you can just throw an exception on the method